pysteps/blending/steps.py

```diff
@@ -560,336 +570,659 @@ def forecast(
         precip_models, bp_filter, decompositor, recompositor, fft, domain
     )
 
-    # 4. Estimate AR parameters for the radar rainfall field
-    PHI = _estimate_ar_parameters_radar(
-        precip_cascade, ar_order, n_cascade_levels, MASK_thr
-    )
+    # 2.3 Check for zero input fields in the radar and NWP data.
+    zero_precip_radar = blending.utils.check_norain(precip, precip_thr, norain_thr)
+    # The norain fraction threshold used for nwp is the default value of 0.0,
+    # since nwp does not suffer from clutter.
+    zero_model_fields = blending.utils.check_norain(precip_models_pm, precip_thr)
```

Comment:
I have noticed that there can be some artifacts when there is very little rainfall, so I would suggest to use the same norain_thr for the NWP model fields.

Reply:
was this resolved in the meantime?

Reply:
Yes, I added it in my commit.